app/renderer/ui/components/walletForm/steps/walletSeedTypeSelection/index.tsx

```diff
@@ -7,8 +7,9 @@ const styles = require("./style.scss")
 
 export interface Iprops {
   className?: string
-  nextStep: { newSeed: () => Promise<void> }
-
+  nextStep: {
+    newSeed: () => Promise<void>,
+    newSeedPrefilled: () => Promise<void> }
```

Comment:
Please move '}' character in line 12 to next line for better alignment :sweat_smile: